terraform/scripts/foundation.sh

```diff
@@ -5,10 +5,14 @@
 
 set -eux
 
-sudo mount /dev/xvdf /mnt
-echo '/dev/xvdf /hab     ext4   defaults 0 0' | sudo tee -a /etc/fstab
-sudo mkdir -p /mnt/hab
-sudo ln -s /mnt/hab /hab
+if mountpoint -q /mnt; then
+    echo "Already Mounted"
```

Comment:
What's the situation where this would already be mounted?

Reply:
Same reason for the force above. If provisioning fails for some other reason, it will re-run the null resource and fail

Reply:
Yeah this feels like the wrong strategy as well...

Reply:
Same answer as above on force.